pilkit/processors/base.py

```diff
@@ -110,17 +110,23 @@ def process(self, img):
         return composite
 
 
+# fix deprecation warnings for PIL
+try:
+    from PIL.Image import Transpose as PIL_TRANSPOSE
+except ImportError:
+    PIL_TRANSPOSE = Image.TRANSPOSE
```

Comment:
It doesn't look like older versions of Pillow have a `TRANSPOSE` constant in the `Image` module. It's just `Image.<CONSTANT>`. So should this just read `PIL_TRANSPOSE = Image`?

Reply:
In addition, it seems `pilkit` tries to keep Pillow/PIL compatibility fixes contained in `pilkit/lib.py`, so this maybe needs to be moved there and then this module can import the `PIL_TRANSPOSE` from there?

(note I'm not a maintainer of this package, so this is just a suggestion)

Reply:
Thanks for the feedback on this! Hoping to get to it soonish :)